src/importer/importer.js

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import csvtojson from 'csvtojson';
+
+export class Importer{
+    constructor(dirwatcher){
+        this.dirwatcher = dirwatcher;
+        this.path = 'src/data';
+
+        this.dirwatcher.watch(this.path, 1000);
+        this.dirwatcher.myEmitter.on('changed', (filename) => {
+            this.import(this.path+'/'+filename).then(jsonObj => console.log(jsonObj));
+
+            this.importSync(this.path+'/'+filename).then(jsonObj => console.log(jsonObj));
```

Comment:
Hm, if it's syncronous, why do it have `.then()` like async Promises do?

Reply:
Sorry it was overlooked. Corrected now